CHANGELOG.md

```diff
@@ -4,6 +4,79 @@
 
 # JupyterLab Changelog
 
+## v4.3 (beta)
+
+JupyterLab 4.3 includes a number of new features (described below), bug fixes, and enhancements.
+This release is compatible with extensions supporting JupyterLab 4.0.
+Extension authors are recommended to consult the [Extension Migration Guide](https://jupyterlab.readthedocs.io/en/latest/extension/extension_migration.html#jupyterlab-4-2-to-4-3) which lists deprecations and changes to the public API.
+
+### Minimap
+
+The virtual scrollbar now acts as a minimap and exposes information on whether a cell:
+
+- is running/scheduled (`[*]`), idle (`[ ]`) or was already run (e.g. `[1]`)
+- was modified since it was last executed ("dirty") - orange background
+- has output contains an error (traceback) - red background
+- is markdown/code (`[ ]`/`[*]`/`[1]` for code, nothing for markdown)
```

Comment:
Should we also mention "raw", for which I assume nothing gets displayed?

Reply:
You are correct, raw type is not differentiated in any way. But then it is not that commonly used so I think it is fine not to mention it.